nucliadb_protos/resources.proto

```diff
@@ -84,7 +84,7 @@ message Origin {
     google.protobuf.Timestamp modified = 5;
     map<string, string> metadata = 6;
     repeated string tags = 7;
-    repeated string colaborators = 8;
+    repeated string collaborators = 8;
```

Comment:
AFAIK this will break actual messages. Should be deprecated ? Lets discuss

Reply:
Yes, this will break things. This reminds me that it'd probably be a good idea to setup protobuf linting to prevent us from introducing breaking changes.

Reply:
I re-added the field.